compiler/rustc_builtin_macros/src/deriving/generic/mod.rs

```diff
@@ -174,6 +174,7 @@
 //! )
 //! ```
 
+use ast::MatchKind;
```

Comment:
I see there are multiple imports autoinserted by rust-analyzer (probably), could you change them all to use the idiomatic style?

Reply:
Yup